rma_sale/controllers/sale_portal.py

```diff
@@ -68,8 +68,11 @@ def request_rma(self, order_id, access_token=None, **post):
         for rec in rma:
             rec.origin += _(" (Portal)")
         # Add the user as follower of the created RMAs so they can
-        # later view them.
-        rma.message_subscribe([request.env.user.partner_id.id])
+        # later view them. Subscribe the user to the notification subtype
+        rma.message_subscribe(
+            partner_ids=[request.env.user.partner_id.id],
+            subtype_ids=request.env.ref("rma.mt_rma_notification").ids,
```

Comment:
With this you are subscribing it only to that subtype, not the rest of the standard ones, so I think this is not the desired effect.

Reply:
I thought the same until I discovered this bad surprise, as I think there's no API for that.